src/libstd/sys/unix/process/process_fuchsia.rs

```diff
@@ -111,12 +110,16 @@ impl Command {
             callback()?;
         }
 
-        let process_handle = mx_cvt(launchpad_start(launchpad))?;
+        let mut process_handle: mx_handle_t = 0;
+        let mut err_msg: *const libc::c_char = ptr::null();
+        mx_cvt(launchpad_go(launchpad, &mut process_handle, &mut err_msg))?;
+        // TODO: See if we want to do something with that err_msg
```

Comment:
The "tidy" linter didn't like this. It feels like a TODO to me, but if it wants FIXME, I suppose we should make it happy.